File: mini-max-sum/mini-max-sum.go
package main

import (
	"fmt"
	"math"
)

func miniMaxSum(arr []int32) {
	var sumMin uint64 = math.MaxUint64
	var sumMax uint64 = 0

	for i := 0; i < len(arr); i++ {
		var sum uint64 = 0
		for j, n := range arr {
			if i != j {
				sum += uint64(n)
			}
		}

		if sum < sumMin {
			sumMin = sum
		}

		if sum > sumMax {
			sumMax = sum
		}
	}

	fmt.Println(sumMin, sumMax)
}

File: mini-max-sum/MiniMaxSum.java
import java.util.List;

public class MiniMaxSum {

  public static void miniMaxSum(List<Integer> ns) {
    long sumMin = Long.MAX_VALUE;
    long sumMax = Long.MIN_VALUE;

    for (int i = 0; i < ns.size(); i++) {
      long sum = 0;

      for (int j = 0; j < ns.size(); j++) {
        if (i != j) {
          sum += ns.get(j);  
        }
      }

      sumMin = Math.min(sumMin, sum);
      sumMax = Math.max(sumMax, sum);
    }

    System.out.println(
      String.format("%s %s", sumMin, sumMax)
    );
  }

}
